index.js

```diff
@@ -181,6 +182,9 @@ class ServerlessPlugin {
           const resolvedLayerArn = arnVersionMap.get(node);
           if (resolvedLayerArn) {
             this.update(resolvedLayerArn);
+            // Avoid logging the same resolved layer multiple times
+            if (self.resolvedLayers.some(item => item === resolvedLayerArn)) return
```

Comment:
Then this can be expressed as more cleaner way :)

```suggestion
            if (self.resolvedLayers.has(resolvedLayerArn)) {
              return;
            }
            self.resolvedLayers.add(resolvedLayerArn);
```

Reply:
And done